extractor/src/main/java/org/schabi/newpipe/extractor/services/media_ccc/linkHandler/MediaCCCStreamLinkHandlerFactory.java

```diff
@@ -1,16 +1,39 @@
 package org.schabi.newpipe.extractor.services.media_ccc.linkHandler;
 
+import org.schabi.newpipe.extractor.exceptions.FoundAdException;
 import org.schabi.newpipe.extractor.exceptions.ParsingException;
 import org.schabi.newpipe.extractor.linkhandler.LinkHandlerFactory;
+import org.schabi.newpipe.extractor.utils.Utils;
+
+import java.net.MalformedURLException;
+import java.net.URL;
 
 public class MediaCCCStreamLinkHandlerFactory extends LinkHandlerFactory {
 
     @Override
-    public String getId(String url) throws ParsingException {
-        if(url.startsWith("https://api.media.ccc.de/public/events/") &&
-            !url.contains("?q=")) {
-            return url.replace("https://api.media.ccc.de/public/events/", "");
+    public String getId(String urlString) throws ParsingException {
+        if (urlString.startsWith("https://api.media.ccc.de/public/events/") &&
+                !urlString.contains("?q=")) {
+            return urlString.replace("https://api.media.ccc.de/public/events/", "");
```

Comment:
Wouldn't it be a better idea to use substring? That's what we want, in the end

Reply:
You can notice I didn't changed anything here, just the parameter name.
Anyway, I've done it, if I understood your request